setup.py

```diff
@@ -23,7 +23,7 @@
         'numpy>=1.6.1',
         'SciPy>= 0.9',
         'theano>=0.8.2',
-        'keras>=1.0.6',
+        'keras>=2.2.4',
```

Comment:
Related to above comment - is it intentional to not support Keras < 2.2.4? Isn't this too restrictive?